frontend/src/metabase/components/ItemsTable/DefaultItemRenderer.tsx

```diff
@@ -0,0 +1,90 @@
+import { useCallback } from "react";
```

Comment:
This is moved from `frontend/src/metabase/components/ItemsTable/BaseItemsTable/BaseItemsTable.tsx`